modules/apps/app-builder/app-builder-lang/src/main/resources/content/Language.properties

```diff
@@ -110,4 +111,6 @@ untitled-table-view=Untitled Table View
 widget=Widget
 x-of-x-entries={0} of {1} Entries
 x-of-x-entry={0} of {1} Entry
-you-do-not-have-the-permission-to-manage-this-entry=You do not have the permission to manage this entry.
\ No newline at end of file
+you-do-not-have-the-permission-to-manage-this-entry=You do not have the permission to manage this entry.
+you-dont-have-access-to-this-app-sign-in-to-access-it=You don't have access to this app. Sign in to access it.
+you-dont-have-permissions-to-access-this-app-contact-the-app-administrator-to-request-the-access=You don't have permissions to access this app. Contact the app administrator to request the access.
```

Comment:
Hey @kevenleone, usually Bchan doesn't like contraction. You could replace "don't" with "do not"?

Reply:
Sure, @albinojunior

Thanks for the advice